### PR TITLE
Test new embedding

### LINKED ISSUE #38: Define Test set for evaluation

### PR DESCRIPTION
**Pull request by marshuang80**
## PR overview
This PR includes the scripts and constants used to evaluate our nearest neighbor class. The `test_nn.py` file performance a bootstrap confidence interval test for NN and outputs sensitivity as an evaluation metrics. The `test_constants.py` file includes the testcases used for evaluation. 

closes #38 #83

### CODE REVIEW

File: module/tests/test_constants.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+DIABETES = {
+    'ID': 'Q3025883',  
+    'Name': 'type-2 diabetes',
+    'Link': 'https://www.wikidata.org/wiki/Q3025883',
+    'DrugIDs':['Q27275212', 'Q772735', 'Q409898', 'Q15269678',
+               'Q899036', 'Q27077223', 'Q15269682', 'Q954845'
+               'Q19484', 'Q288280'],
+    'DrugNames': ['luseogliflozin', 'miglitol', 'dapagliflozin', 'teneligliptin'
+                  'Colesevelam', 'ertugliflozin', 'anagliptin', 'statin',
```
Comment (villanuevab) [inline module/tests/test_constants.py:50]: Does capitalization matter? Should we convert everything to lowercase when comparing entities?

Comment (marshuang80) [reply module/tests/test_constants.py:50]: These are the names I got from WikiData, I wanted to keep their format consistent. We use the IDs for comparison anyways.

Comment (villanuevab) [reply module/tests/test_constants.py:50]: Gotcha. Thanks!

**Comment by villanuevab** [comment]
We may want to remove 04a076e from this PR, but I don't think it should be an issue. Otherwise it looks good to me! Thanks.

**Comment by marshuang80** [comment]
Just to confirm, is reverting the same as removing?

**Pull request merged**